ruby-2.6.3-circleci/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM parkwhiz/rails-base:2.6.3_20190429154415
+MAINTAINER ParkWhiz <dev@parkwhiz.com>
+
+WORKDIR /
+
+
+# ---------------------
+# -- Just `apk` things.
+# ---------------------
+
+RUN \
+  apk update && \
+  apk upgrade && \
+  apk add docker
```

Comment:
could you add nano as well, this makes it possible to do a little more troubleshooting on circle easier. I don't even think they have vi by default

Reply:
otherwise this is looking real nice.